sklearn/tests/test_base.py

```diff
@@ -1048,6 +1049,7 @@ def test_param_is_non_default(default_value, test_value):
         ((1, 2, 3), [1, 2, 3]),
         ((1, 2, 3), np.array([1, 2, 3])),
         (np.nan, np.nan),
+        (np.nan, pd.NA),
```

Comment:
I don't think that we want to treat `pd.NA` as being the same as `np.nan`.

Reply:
So, should `pa.NA` be treated as non-default?

Reply:
yes

Reply:
To clarify my previous comment: 
If I try to import `pandas` in `test_base.py` in order to test for a param `pd.NA`, the ci fails. 
I think that pandas is not in the test environment. Suggestions welcomed.